pkg/operator/ceph/object/controller.go

```diff
@@ -319,3 +329,34 @@ func updateStatus(client client.Client, name types.NamespacedName, status string
 	}
 	logger.Debugf("object store %q status updated to %q", name, status)
 }
+
+func (r *ReconcileCephObjectStore) handleDelete(objectstore *cephv1.CephObjectStore) error {
```

Comment:
Is this method serving a real purpose? What about just calling `verifyObjectBucketCleanup` in L211?

Reply:
removed `handleDelete` function.